cfpb/consumerfinance.gov
# Archiving: Add non-navigable archive breadcrumb

Adds a non-navigable `Archive` breadcrumb to archived pages.


## Additions

- Adds `Archive` breadcrumb if page has been marked as archived


## How to test this PR

1. Check out branch
2. In admin, mark page as archived
3. Navigate to page and note `Archive` breadcrumb


## Screenshots

<img width="680" alt="Screen Shot 2020-09-27 at 5 23 18 PM" src="https://user-images.githubusercontent.com/778171/94379614-71ffce00-00e6-11eb-8362-62142e0175cd.png">
